lib/iris/fileformats/netcdf.py

```diff
@@ -1858,54 +1881,61 @@ def _create_cf_data_variable(self, cube, dimension_names, local_keys=None,
             An interable of cube attribute keys. Any cube attributes
             with matching keys will become attributes on the data variable.
 
+        * packing (type or string or dict or list): A numpy integer datatype
```

Comment:
It seems a bit fragile to repeat all this between `__init__` and `_create_cf_data_variable`.
Especially as this is not a public docstring for Sphinx, I think it would make more sense to cross-refer to the other here. How about just "`packing` and `fill_value` controls are applied to the data, as described in `__init__`."